packages/fbtee/src/list.tsx

```diff
@@ -0,0 +1,134 @@
+            <fbt:param name="previous items">{output}</fbt:param> &bull;{' '}
+            <fbt:param name="following items">{items[index]}</fbt:param>
+          </fbt>
+        );
+        break;
+      default:
+        output = (
+          <fbt desc="A list of items of various types. {previous items} and {following items} are themselves lists that contain one or more items.">
+            <fbt:param name="previous items">{output}</fbt:param>
+            {', '}
+            <fbt:param name="following items">{items[index]}</fbt:param>
+          </fbt>
+        );
+    }
+  }
+
+  switch (conjunction) {
+    case 'and':
+      return (
+        <fbt desc='A list of items of various types, for example: "item1, item2, item3 and item4"'>
+          <fbt:param name="list of items">{output}</fbt:param>
+          and
+          <fbt:param name="last item">{lastItem}</fbt:param>
+        </fbt>
+      );
+
+    case 'or':
+      return (
+        <fbt desc='A list of items of various types, for example: "item1, item2, item3 or item4"'>
+          <fbt:param name="list of items">{output}</fbt:param>
+          or
+          <fbt:param name="last item">{lastItem}</fbt:param>
+        </fbt>
+      );
+
+    case 'none':
+      switch (delimiter) {
+        case 'semicolon':
+          return (
+            <fbt desc='A list of items of various types, for example: "San Francisco; London; Tokyo". {previous items} itself contains one or more items.'>
+              <fbt:param name="previous items">{output}</fbt:param>
+              {'; '}
+              <fbt:param name="last item">{lastItem}</fbt:param>
+            </fbt>
+          );
+        case 'bullet':
+          return (
+            <fbt desc='A list of items of various types separated by bullets, for example: "San Francisco \u2022 London \u2022 Tokyo". {previous items} contains one or more items.'>
+              <fbt:param name="list of items">{output}</fbt:param> &bull;{' '}
+              <fbt:param name="last item">{lastItem}</fbt:param>
+            </fbt>
+          );
+        default:
+          return (
+            <fbt desc='A list of items of various types, for example: "item1, item2, item3, item4"'>
+              <fbt:param name="list of items">{output}</fbt:param>
+              {', '}
+              <fbt:param name="last item">{lastItem}</fbt:param>
+            </fbt>
+          );
+      }
+    default: {
+      conjunction satisfies never;
+      throw new Error(
+        `Invalid conjunction ${conjunction} provided to '<fbt:list>'.`,
+      );
+    }
+  }
+}
+
+export function List({
+  conjunction,
+  delimiter,
+  items,
+}: {
+  conjunction?: Conjunction;
+  delimiter?: Delimiter;
+  items: Array<string | React.ReactElement | null | undefined>;
+}) {
+  return list(items, conjunction, delimiter);
+}
```

Comment:
Arguably this module could be changed to only export `List` and remove `list`. However, I want to keep a distinction between mostly strings (`list`) and React component (`List`) usage so that the migration to real React components will be easier in the future.